doc/news/changes/minor/20190815PeterMunch

```diff
@@ -0,0 +1,3 @@
+New: Add method to get the coarse-grid cell from CellID.
+<br>
+(Peter Munch, 2019/08/15)
```

Comment:
You want to drop this commit from this pull request.

Reply:
@bangerth I have drop it.